wgengine/userspace.go

```diff
@@ -741,6 +755,17 @@ func (e *userspaceEngine) updateActivityMapsLocked(trackDisco []tailcfg.DiscoKey
 	e.tundev.SetDestIPActivityFuncs(e.destIPActivityFuncs)
 }
 
+func hasOverlap(aips, rips []netaddr.IPPrefix) bool {
```

Comment:
comment that this is different than

https://pkg.go.dev/inet.af/netaddr#IPPrefix.Overlaps
https://pkg.go.dev/inet.af/netaddr#IPSet.Overlaps

This is not about any partial overlap.

Reply:
Done